manifests/event_management.pp

```diff
@@ -12,6 +12,8 @@
   Optional[Integer[0, 5]] $pending_corrective_changes_event_severity  = 2,
   Optional[Integer[0, 5]] $pending_intentional_changes_event_severity = 1,
   Optional[Integer[0, 5]] $no_changes_event_severity                  = 1,
```

Comment:
Nice, it seems like it's just as easy to configure which facts are send for events as well.